@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/performance/fn_perfData.sqf

```diff
@@ -56,9 +56,9 @@ switch (fza_ah64_sfmplusEnvironment) do {
 		_baseFAT = 25.0;
 	};
 };
-private _pa      = round ((_baseAlt + _baroAlt) / 10) * 10;  //feet
+private _pa      = round ((_baseAlt + _baroAlt) / 10) * 10; //feet
 _heli setVariable ["fza_sfmplus_PA", _pa];
-private _fat     = _baseFAT - round((_baroAlt / 1000) * 2);//ambientTemperature select 0; //deg C
+private _fat     = _baseFAT - round((_baroAlt / 1000) * 2);	//ambientTemperature select 0; //deg C
```

Comment:
There are some changes to spaces here which don't need to exist. Please can you correct it?

Reply:
Fixed

Reply:
Fixed